### PR TITLE
Fix spec

### PR DESCRIPTION
**Pull request by astrozzc**
The doc file has mismatch with the actural api respons.

### CODE REVIEW

File: docs/source/specs/openapi.json [hunk 1]
```diff
@@ -613,7 +613,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/GroupWithPrincipalsAndRoles"
+                  "$ref": "#/components/schemas/GroupRolesPagination"
```
Comment (coderbydesign) [inline docs/source/specs/openapi.json]: I think this will just have `RoleOut`, not `GroupRolesPagination`, won't it?

**Comment by catastrophe-brandon** [comment]
Looks good to me!

File: docs/source/specs/openapi.json (hunk 1 above)
Comment (catastrophe-brandon) [reply docs/source/specs/openapi.json]: If we change it to RoleOut, let me know. For the sake of charging forward with test automation I used GroupRolesPagination when I generated my most recent client source.

Comment (coderbydesign) [reply docs/source/specs/openapi.json]: Yeah, I don't think we're returning the paginated schema for a POST response. We're just returning the `RoleOut` schema, unless I'm mistaken.

Comment (astrozzc) [reply docs/source/specs/openapi.json]: Right, it will return a list of RoleOut. Just updated 👍

**Pull request merged**